### PR TITLE
Run Windows FV tests

### PR DESCRIPTION
**Pull request by lmm**
## Description

This PR adds Windows FV tests. The FV test infra is mostly the same as in cni-plugin and the other felix repo. A minor improvement is that these FVs run against VXLAN and bgp clusters.

## Todos
- [ ] Unit tests (full coverage)
- [ ] Integration tests (delete as appropriate) In plan/Not needed/Done
- [ ] Documentation
- [ ] Backport
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: fv/winfv/policy_test.go [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+	_, _, err := powershell(cmd)
+	return err
+}
+
+var _ = Describe("Windows policy test", func() {
+	var (
+		porter, client, clientB, nginx, nginxB string
+	)
+
+	BeforeEach(func() {
```
Comment (song-jiang) [inline fv/winfv/policy_test.go:70]: Worth adding comments that pods/policies are created by infra.

**Review by song-jiang — COMMENTED** [review]
Looks good. Just one comment

File: fv/winfv/policy_test.go (hunk 1 above)
Comment (lmm) [reply fv/winfv/policy_test.go:70]: Done

Comment (song-jiang) [reply fv/winfv/policy_test.go:70]: I think more comments about network policies setup are needed so people can understand test cases. For example 
``` 
Three network polices has been installed by test infra
1. Allow cient to nginx 
2
3
````

**Review by song-jiang — APPROVED** [review]
LGTM

**Pull request merged**